recipes/Cm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cm" %}
+{% set version = "9.0.0" %}
+{% set versions = version.split('.') %}
```

Comment:
This line is unused and should be removed.

Reply:
I have removed this line.